codecov.yml

```diff
@@ -1,18 +1,28 @@
 codecov:
-  require_ci_to_pass: true
-
-comment:
-    behavior: default
-    branches:
-        - "master"
+  require_ci_to_pass: no
 
 coverage:
+  precision: 2
+  round: down
+  range: "70...100"
   status:
     project:
       default:
+        # basic
         target: auto
-        threshold: 0%
-
+        threshold: 1%
+        base: auto
     patch:
       default:
-        target: 100%
+        # basic
+        target: 50%
+        threshold: 1%
+        base: auto
```

Comment:
We need 100% coverage for newly added code.  While this may be annoying for some forms of refactoring, even in that context it's at least prompting us to check and make sure we didn't actually reduce coverage while refactoring.